src/modules/feedback/components/FeedbackBoxContainer.js

```diff
@@ -44,15 +44,19 @@ type PropsType = {|
   extras: ?Array<ExtraModel>,
   postFeedbackDataOverride?: FeedbackParamsType => void,
   closeFeedbackModal: () => void,
-  onSubmit: () => void,
+  onSubmit: (value?: string) => void,
   t: TFunction,
-  theme: ThemeType
+  theme: ThemeType,
+  feedbackSent: string
 |}
 
+type SendingStatusType = 'IDLE' | 'SUCCESS' | 'ERROR'
+
 type StateType = {|
   feedbackOptions: Array<FeedbackVariant>,
   selectedFeedbackOption: FeedbackVariant,
-  comment: string
+  comment: string,
+  feedbackSent: SendingStatusType
```

Comment:
No need to define this again, it's a duplication of state. Instead, you should pass `feedbackSent` of `FeedbackModal`'s state down to this component as prop.

Reply:
Will use and handle the prop and its value of `feedbackSent` now. But to declare the param and type of  prop `onSubmit: (feedbackSent?: SendingStatusType) => void`, isn't it needful to set the type here?